addons/mail/static/src/components/composer/composer.js

```diff
@@ -178,6 +186,17 @@ class Composer extends Component {
         this.trigger('o-message-posted');
     }
 
+    /**
+     * @private
+     */
+    _update() {
+        // FIXME this would not be needed if input binding is handled in a new component
+        // (task-2284290)
```

Comment:
Would drop this comment. Looks useful for future plan, but it's not really a FIXME, more a refactoring for improving architecture of code with common code in managing subject & text/main inputs of composer. Task by itself is enough, not need to refer in code if it ain't broke.